www/js/lib/popovers.js

```diff
@@ -111,7 +111,12 @@ function cleanUpLedeContent (node) {
         // The reason we prefer innerText is that it strips out hidden text and unnecessary whitespace, which is not the case with textContent
         const innerText = para.innerText ? para.innerText : para.textContent;
         const text = innerText.trim();
-        return !/^\s*$/.test(text) && text.length >= 50;
+
+        // removing the para with less than 50 characters
+        // regex to check the paragraph if its too short or a brief description
+        const briefDescriptionRegex = /^.{1,100}$/;
```

Comment:
I don't think this is a safe way to filter out these short descriptions. We don't know in advance the amount of text, so setting a limit of 100 characters seems arbitrary and will likely have unintended consequences elsewhere, for example filtering out legitimate paragraphs with only 100 characters in them.

Reply:
The reason we do have a filter requiring at least 50 characters is that there are lots of small empty paragraphs or empty paragraphs with things like `<span>&nbsp;</span>` which can't be filtered out any other way. Through testing, we determined that 50 characters caught nearly all these while never removing any nodes with meaningful text.

Reply:
Got it Sir! i will change the logic.